app/views/catalog/access_panels/_online.html.erb

```diff
@@ -11,8 +11,10 @@
     <h3>
       <% if document.is_a_database? %>
         Search this database
-      <% else %>
+      <% elsif document.druid %>
```

Comment:
We use similar logic in the search results view: 
https://github.com/sul-dlss/SearchWorks/blob/37f7c748a0b9fb576e0dff9c77c68c064c3d7395/app/views/catalog/_index_online_section.html.erb#L2-L6